ChargingCurrentReadings.c

```diff
@@ -0,0 +1,24 @@
+#include "ChargingCurrentReadings.h"
+
+int arr[3] = {0, 0, 0};
```

Comment:
using global variables need careful considerations as it maintains state throughout the lifetime and hence requires proper handling.